### PR TITLE
dotCMS/core#17901 Fixing some details

### PR DESCRIPTION
**Pull request by freddyucv**
Fixing: https://github.com/dotCMS/core/issues/17901#issuecomment-601354433

### CODE REVIEW

File: projects/dot-rules/src/lib/styles/rule-engine.scss [hunk 1]
```diff
@@ -94,6 +94,15 @@ $finder-s-input: '.ui.input input';
         height: 100%;
     }
 
+    .close-button {
+        font-size: 14px;
```
Comment (alfredo-dotcms) [inline projects/dot-rules/src/lib/styles/rule-engine.scss]: image

shouldn't you create a var for that?

File: projects/dot-rules/src/lib/rule-engine.container.ts [hunk 2]
```diff
@@ -22,6 +22,7 @@ import { BundleService, IPublishEnvironment } from './services/bundle-service';
 import { ActivatedRoute, Params } from '@angular/router';
 import { HttpCode } from 'dotcms-js';
 import { LoggerService } from 'dotcms-js';
+import { RuleViewService } from './services/dot-view-rule-service';
 
 // tslint:disable-next-line:no-unused-variable
```
Comment (alfredo-dotcms) [inline projects/dot-rules/src/lib/rule-engine.container.ts]: remove?

File: projects/dot-rules/src/lib/rule-engine.ts [hunk 3]
```diff
@@ -120,20 +122,26 @@ export class RuleEngineComponent {
...
-    constructor(resources: I18nService) {
+    constructor(resources: I18nService, private ruleViewService: RuleViewService) {
         this.resources = resources;
         resources.get(I8N_BASE).subscribe(rsrc => {});
         this.filterText = '';
         this.rules = [];
         this._rsrcCache = {};
         this.status = null;
+
+        this.ruleViewService.message.subscribe(message => {
```
Comment (alfredo-dotcms) [inline projects/dot-rules/src/lib/rule-engine.ts]: tyoe?

File: projects/dot-rules/src/lib/styles/rule-engine.scss (hunk 1 above)
Comment (fmontes) [reply projects/dot-rules/src/lib/styles/rule-engine.scss]: Not sure if we can have variables here.

File: projects/dot-rules/src/lib/rule-engine.container.ts (hunk 2 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/rule-engine.container.ts]: done https://github.com/dotCMS/core-web/pull/1289/commits/4df18ab78c2446282bbbd4d0bcf781cf39d4d912#diff-2f81c9d19a8208425bfaa24e0f21564eL28

File: projects/dot-rules/src/lib/rule-engine.ts (hunk 3 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/rule-engine.ts]: done https://github.com/dotCMS/core-web/pull/1289/commits/4df18ab78c2446282bbbd4d0bcf781cf39d4d912#diff-bad0e3f89080e5eca6ec4d92d393469fR142

File: projects/dot-rules/src/lib/styles/rule-engine.scss (hunk 1 above)
Comment (freddyucv) [reply projects/dot-rules/src/lib/styles/rule-engine.scss]: done https://github.com/dotCMS/core-web/pull/1289/commits/4df18ab78c2446282bbbd4d0bcf781cf39d4d912#diff-4a35aea585a8446a8a175a9ec9a43f72R99

**Pull request merged**